clusterloader2/pkg/prometheus/prometheus.go

```diff
@@ -142,15 +141,10 @@ func (pc *PrometheusController) SetUpPrometheusStack() error {
 			return err
 		}
 	}
-	if pc.isKubemark() {
-		if err := pc.applyManifests(kubemarkServiceMonitors); err != nil {
-			return err
-		}
-	} else {
-		if err := pc.applyManifests(defaultServiceMonitors); err != nil {
-			return err
-		}
+	if err := pc.applyManifests(defaultServiceMonitors); err != nil {
```

Comment:
This is causing the kubemark tests to fail.

it should be
```
if !pc.isKubemark() {
  if err := pc.applyManifests(defaultServiceMonitors); err != nil {
    return err
  }
}
```

Reply:
Thank you.  Is it means if pc.isKubemark()=true,we don't need to apply defaultServiceMonitors?

Reply:
Yes, if `pc.isKubemark()=true` then we **should not** apply defaultServiceMonitors, otherwise the test will fail.

Reply:
Thank you.I fix it.